serverless/db/mongo/models/contact.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-use-before-define, no-console */
+import { Promise as bbPromise } from 'bluebird';
+import contactSchema from '../schemas/contactSchema';
+
+export default (db) => {
+  /**
+  * Function: sendSupportMailAndNotifySlack
+  *
+  * 1) Construct the emailRequest object for sendRawEmail from user input.
+  * 2) Invoke sendRawEmail from Email schema.
+  * 3) Save the document in Contact collection with messageId from sendRawEmail response.
+  * 4) Invoke notifySlack from Email model.
+  * 5) Resolves || Rejects with result.
+  *
+  * @param {Object} contactForm - {
+      name - {string} Name of the customer filing support
+      userId - {string} mongo ID of user doc
+      message - {string} customer's support message
+      emailAddress - {string} customer's mailId
+      ccUser - {bool} whether to add user to CC this message
+    } - Email JSON containing all the required SES parameters.
+  *
+  * @return {object} - Promise: resolve or reject the response.
+  */
+  contactSchema.statics.sendSupportMailAndNotifySlack = (contactForm, Email)  =>
+  new Promise((resolve, reject) => {
+    console.log('\n\n@Contact.sendSupportMailAndNotifySlack\n');
+    let emailRequest = {
+      sourceEmail: 'support@nj2jp.com',
+      toEmailAddresses: ['support@nj2jp.com'],
+      replyToAddresses: [contactForm.emailAddress],
+      bodyTextData: contactForm.message,
+      bodyTextCharset: 'utf8',
+      subjectData: 'Customer Name: ' + contactForm.name + ' - Support Required!!',
+      subjectCharset: 'utf8'
+    };
+
+    if (contactForm.ccUser) {
+      emailRequest.ccEmailAddresses = [contactForm.emailAddress];
+    }
+
+    if (contactForm.userId) {
+      emailRequest.userId = contactForm.userId;
+    }
+
+    let contactDocument = {};
+    let slackWebhook = '';
+    let slackMessage = '';
+
+    Email.sendRawEmail(emailRequest)
+    .then((response) => {
+      console.log('SUCCEEDED: Email has been sent to NJ2JP support:', response);
+      contactDocument.messageId = response.MessageId;
+
+      slackWebhook = process.env.SLACK_CUSTOMER_WEBHOOK;
+      slackMessage = 'We have got a customer support from ' + contactForm.emailAddress + '. Log into <https://privateemail.com/appsuite/> to answer their query.'
```

Comment:
Let's change the message to `We received a support request from ${contactForm.emailAddress}. Log into....`